core/src/banking_stage/forward_worker.rs

```diff
@@ -157,7 +158,7 @@ mod tests {
             poh_recorder,
             bank_forks,
             cluster_info,
-            Arc::default(),
+            Arc::new(ConnectionCache::new("test")),
```

Comment:
This should have been `connection_cache_test` as in other cases in https://github.com/solana-labs/solana/pull/31803